stage0/run.go

```diff
@@ -481,6 +481,13 @@ func Prepare(cfg PrepareConfig, dir string, uuid *types.UUID) error {
 
 	cfg.CommonConfig.ManifestData = string(pmb)
 
+	// create pod lock file for app add/rm operations.
+	f, err := os.OpenFile(common.PodManifestLockPath(dir), os.O_CREATE|os.O_RDWR, 0600)
+	if err != nil && !os.IsExist(err) {
```

Comment:
isn't the `IsExist` superfluous? You'll only get `EEXIST` if you use `O_EXCL` too.